static/usr/lib/systemd/system/console-conf@.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=Ubuntu Core Firstboot Configuration %I
+After=systemd-user-sessions.service plymouth-quit-wait.service
+After=rc-local.service
+# on core18/core20 the firstboot seeding happens via the core[18].start-snapd service,
+# make sure to start after so that the 'snap' command is available
+After=core18.start-snapd.service core.start-snapd.service
+# on core20 the user may invoke a recovery chooser, make sure the detection
+# service runs before
+After=snapd.recovery-chooser-trigger.service
+IgnoreOnIsolate=yes
+ConditionPathExists=/dev/tty0
+ConditionPathExists=|!/var/lib/console-conf/complete
+ConditionPathExists=|/run/snapd-recovery-chooser-triggered
+StartLimitInterval=0
+Conflicts=getty@%i.service
+
+[Service]
+Environment=PYTHONPATH=/usr/share/subiquity
+ExecStartPre=/bin/systemctl stop getty@%i
+ExecStart=/sbin/agetty -i -n --noclear -l /usr/share/subiquity/console-conf-wrapper %I $TERM
+# start getty only if we're done
+ExecStopPost=/bin/sh -c 'if [ "$EXIT_STATUS" = "0" ]; then /bin/systemctl start getty@%i; fi'
+Type=idle
+Restart=always
+RestartSec=0
+UtmpIdentifier=%I
+TTYPath=/dev/%I
+TTYReset=yes
+TTYVHangup=yes
+TTYVTDisallocate=yes
+KillMode=process
```

Comment:
This needs an explanation. Systemd documentation recommends never using this. So it would be nice to explain why we need it in a comment.

Reply:
I've dropped it